src/main/java/com/slcube/shop/business/member/domain/MemberStatus.java

```diff
@@ -0,0 +1,5 @@
+package com.slcube.shop.business.member.domain;
+
+public enum MemberStatus {
+    SIGN_UP, SIGN_OUT
+}
```

Comment:
```suggestion
public enum MemberStatus {
    MEMBER, EXCLUDED
}
```

와 같이 회원인지 아닌 회원탈퇴인지로 바꾸는게 어떨까요?? 왜냐하면 나중에 admin 케이스 ...etc 를 설정할경우 이름 정하기 애매 해질거같아서요